backend/comment-section.go

```diff
@@ -0,0 +1,173 @@
+// Copyright Google Inc.
```

Comment:
Yes, the Hotel sample has a different logic and I think each sample should be able to specify its own logic. What do you mean "restrict to only the comment form endpoint"? It's handling only requests for "/samples_templates/comment_section/", so I believe it's already restricted.